src/markdown-pages/docs/managers/github-manager.md

```diff
@@ -0,0 +1,290 @@
+@GitHubIssue( issue = "1000")<br>
+Test class SimBankIVT<br>
+Test method testNotNull() result: _Passed_ ![passed icon:](passed.svg)<br>
+Test method checkBankIsAvailable() result: _Failed with defects_ ![failed with defects icon:](failed-with-defects.svg)<br><br>
+<b>Test class result: _Passed with defects_</b> ![passed with defects icon:](passed-with-defects.svg) <br>
+
+
+
+Next steps:<br> 
+The method _checkBankIsAvailable_ returned a result of _Failed With Defects_. As the test did not use the _ContinueOnTestFailure_ annotation, the test would have ended prematurely. However, as the _checkBankIsAvailable_ method was the last test method, the results of all test methods in the class are known. All methods that are expected to pass returned a result of _Passed_, so the overall result of the test is _Passed With Defects_. No further investigation required.
+
+### Scenario 5:
+
+Scenario 5 uses the _@GitHubIssue_ annotation on the test class _SimBankIVT_ without the _@ContinueOnTestFailure_ annotation:
+
+@Test<br>
+@GitHubIssue( issue = "1000")<br>
+Test class SimBankIVT<br>
+Test method testNotNull() result: _Failed with defects_ ![failed with defects icon:](failed-with-defects.svg)<br>
+Test method checkBankIsAvailable() result: _Unknown_<br><br>
+<b>Test class result: _Failed with defects_</b> ![failed with defects icon:](failed-with-defects.svg)<br>
+
+Next steps:<br> 
+The _testNotNull_ method returned a status of _Failed With Defects_. As the _ContinueOnTestFailure_ annotation is not used, the results of the following methods - in this example _checkBankIsAvailable_ are not known. The overall result returned is _Failed With Defects_. To check the result of all methods, add the _@ContinueOnTestFailure_ annotation for future test runs.
+
+
+# <a name="configuring"></a>Configuring 
+
+You must configure your CPS properties so that the GitHub Manager knows the GitHub instance in which to look for the specified issue. The GitHub Manager converts the GitHub instance URL that is provided in the CPS to the correct URL for the GitHub API, removing the need to manually supply a URL for the GitHub API. Alternatively, you can set the GitHub repository in the _@GitHubIssue_ annotation. The value in the annotation takes precedence over the value in the CPS. 
+
+## <a name="cps"></a>Configuration Properties
+
+The following are properties used to configure the GitHub Manager:
+
+<details>
+<summary>GitHub Instance URL CPS Property</summary>
+
+| Property: | GitHub Instance URL CPS Property |
+| --------------------------------------- | :------------------------------------- |
+| Name: | githubissue.instance.url |
+| Description: | The GitHub instance for the issue  |
+| Required:  | No |
+| Default value: | https://github.com |
+| Valid values: | A valid GitHub instance, for example, <br> _https://github.ibm.com_  |
+| Examples: | <code>githubissue.instance.DEFAULT.url=https://github.com<br> githubissue.instance.IBM.url=https://github.ibm.com </code> |
```

Comment:
can we not use GitHub.ibm.com as an example URL please

Reply:
have substituted with an example company name